docs/dev_101_services.md

```diff
@@ -59,20 +59,33 @@ Services are published under the domain name of your integration, so in `service
 ```yaml
 # Example services.yaml entry
 
+# Service ID
 set_speed:
+  # Service name as shown in UI
+  name: Set fan speed.
   # Description of the service
   description: Sets fan speed.
+  # If the service accepts entity IDs, target allows the user to specify entities by entity, device, or area. If `target` is specified, `entity_id` should not be defined in the `fields` map.
+  target:
+    # Target supports the entity, device, and area selectors (https://www.home-assistant.io/docs/blueprint/selectors/). Entity selector parameters will automatically be applied to device and area, and device selector parameters will automatically be applied to area. In this example, the lists will be filtered by: entities in the fan domain, devices that are linked to entities in the fan domain, and areas that have entities in the fan domain.
+    entity:
+      domain: fan
```

Comment:
This is a bit of a misleading example, as this is the default 😄 

By default is shows only targets matching entities from the same domain as the service.

Reply:
OK I can remove that since I linked to the docs on selectors

Reply:
Or should I use a custom service like `hue_activate_scene`

Reply:
actually nvm, that's not an entity service

Reply:
Maybe just make it an obvious fake example? So it doesn't matter?

Reply:
`By default is shows only targets matching entities from the same domain as the service.` if this statement is true, why did we need to add an `integration: zwave_js` filter to `set_config_parameter`?

Reply:
`By default is shows only targets matching entities from the same domain as the service.` if this statement is true, why did we need to add an `integration: zwave_js` filter to `set_config_parameter`?